src/__tests__/cli.ts

```diff
@@ -437,3 +437,75 @@ describe('cli integration tests with init option', () => {
     });
   });
 });
+
+describe('cli integration tests with ignore file', () => {
```

Comment:
Is this setup and test different the base `cli integration tests`?

(I'm on my phone, but it looks the same to me)

Reply:
I've combined the new scenarios to the existing `cli integration tests` setup.